src/components/ProgressBars/LinearProgressBar/Bar/Bar.module.scss

```diff
@@ -10,19 +9,19 @@
 }
 
 .typePrimaryPrimary {
-  @include theme-prop(background-color, primary-color);
+  background-color: var(--primary-color);
 }
 
 .typePrimarySecondary {
-  @include theme-prop(background-color, dark-color);
+  background-color: var(--color-mud_black);
```

Comment:
I really sure we have semantic token for this (even without the themeprop mixin)

Reply:
<img width="633" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/aad89e7d-ec15-408e-ae16-61a61bd4399b">
<img width="716" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/b388d3a8-10c0-4d92-8520-4c0857e2d453">

Reply:
Replaced with `--inverted-color-background`